packages/eslint-plugin/README.md

```diff
@@ -0,0 +1,44 @@
+<div align="center">
+
+# @sumup/eslint-plugin
```

Comment:
I'd give this a more design-system-specific name. Maybe `@sumup/circuit-ui-eslint-plugin`?

Reply:
That works too! By convention it would be `@sumup/eslint-plugin-circuit-ui` though. But it would be added to configs like `plugins: ["@sumup/circuit-ui"]` (yep confusing).

It's true that it might make sense to keep the "@sumup" name free in case we have more custom plugins in the future

Reply:
Done in e858eafb02951614e622e7dc073dd5082c404662